ci-scripts/run_chain_abci.sh

```diff
@@ -8,10 +8,17 @@ if [ x"${SGX_MODE}" == "xHW" ]; then
   LD_LIBRARY_PATH=/opt/intel/libsgx-enclave-common/aesm /opt/intel/libsgx-enclave-common/aesm/aesm_service &
 
   echo "[aesm_service] Running in background ..."
-  # Wait for aesm_service to initialize
-  sleep 4
```

Comment:
Not sure is the original sleep added for a reason, @linfeng-crypto ?

Reply:
This has existed quite a while but the sleep time has been increased since then. Perhaps we should use other ways to make sure aesm service is ready.
https://github.com/crypto-com/chain-tx-enclave/blob/8e1ecad29eed5cf3ac702d260bfdf67b0f186eb8/tx-validation/entrypoint.sh

Reply:
But in the drone integration tests pipeline, I never added any sleep, and have no problem so far.